src/pages/SubmitWorkflow.js

```diff
@@ -24,12 +25,18 @@ export const SubmitWorkflow = () => {
       setCbasStatus(cbasStatus)
     }
 
+    const loadRunsData = async () => {
+      const runs = await Ajax(signal).Cbas.runs.get()
+      setRunsData(runs.runs)
```

Comment:
Would it make sense to add error handling in case the request fails?

Reply:
Reassigned to a follow up ticket

Reply:
Added this to AC of [BW-1319](https://broadworkbench.atlassian.net/browse/BW-1319)